configs/mcli/olmo7-ablation.yaml

```
@@ -0,0 +1,36 @@
+name: olmo7-ablation  # can't have "_" or "." here
+image: mosaicml/pytorch:2.1.2_cu121-python3.10-ubuntu20.04
+compute:
+  gpus: 64
+  cluster: r12z3
```

This should be `r7z2` now (see message from MosaicML folks on Slack).
```suggestion
  cluster: r7z2
```

---

I've been switching back and forth depending on what's available.